source/install/desktop.rst

```diff
@@ -71,42 +73,42 @@ Linux
 Generic Linux package
 ~~~~~~~~~~~~~~~~~~~~~
 
-1. Download latest version of the Mattermost desktop app:
+1. Download the latest version of the Mattermost desktop app:
 
-  64-bit systems:
+ - 64-bit systems:
    `mattermost-desktop-4.5.0-linux-x64.tar.gz <https://releases.mattermost.com/desktop/4.5.0/mattermost-desktop-4.5.0-linux-x64.tar.gz>`__
-  32-bit systems:
+ - 32-bit systems:
    `mattermost-desktop-4.5.0-linux-ia32.tar.gz <https://releases.mattermost.com/desktop/4.5.0/mattermost-desktop-4.5.0-linux-ia32.tar.gz>`__
 
 2. Extract the archive to a convenient location. You can then execute ``mattermost-desktop``, which is located inside the extracted directory.
 
-3. To create a Desktop launcher, open the file ``README.md`` and follow the instructions in the *Desktop launcher* section.
+3. To create a Desktop launcher, open the file **README.md** and follow the instructions in the **Desktop launcher** section.
 
 Ubuntu and Debian-based systems
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Unofficial, community-driven .deb packages are available.
+Unofficial, community-driven ``.deb`` packages are available.
 
-1. Download the latest version of the Mattermost desktop app:
+1. Download the latest version of the Mattermost Desktop App:
 
-  64-bit systems
+ - 64-bit systems
    `mattermost-desktop-4.5.0-linux-amd64.deb <https://releases.mattermost.com/desktop/4.5.0/mattermost-desktop-4.5.0-linux-amd64.deb>`__
-  32-bit systems
+ - 32-bit systems
    `mattermost-desktop-4.5.0-linux-i386.deb <https://releases.mattermost.com/desktop/4.5.0/mattermost-desktop-4.5.0-linux-i386.deb>`__
 
 2. At the command line, execute one of the following commands depending on the package that you downloaded:
 
-  64-bit systems
+ - 64-bit systems
     ``sudo dpkg -i mattermost-desktop-4.5.0-linux-amd64.deb``
-  32-bit systems
+ - 32-bit systems
     ``sudo dpkg -i mattermost-desktop-4.5.0-linux-i386.deb``
 
-3. To run Mattermost, open **Dash** (located at top left corner) and input ``mattermost``, then click the Mattermost icon.
+3. To run Mattermost, open **Dash** (located at the top left corner), enter **mattermost**, then click the Mattermost icon.
```

Comment:
For some reason the formatting for the word "mattermost" looks weird in preview, but not sure why.